OpenRA.Mods.Common/Traits/Modifiers/FrozenUnderFog.cs

```diff
@@ -87,19 +93,21 @@ public void Tick(Actor self)
 			VisibilityHash = 0;
 			foreach (var player in self.World.Players)
 			{
-				bool isVisible;
 				FrozenActor frozenActor;
+				bool isVisible;
 				if (!initialized)
```

Comment:
Player actors are constructed in the `World` constructor through `ICreatePlayers.CreatePlayers` while `SpawnMapActors.WorldLoaded` is called from `World.LoadComplete` which, in turn, is called by `Game.StartGame` after it creates a new `World`.